komfydb/storage/catalog.cc

```diff
@@ -0,0 +1,79 @@
+#include "absl/container/flat_hash_map.h"
+#include "absl/status/statusor.h"
+
+#include "komfydb/storage/catalog.h"
+#include "komfydb/utils/status_macros.h"
+#include "komfydb/utils/utility.h"
+
+namespace {
+
+template <typename K, typename V>
+absl::StatusOr<V> StatusOrMapElement(const absl::flat_hash_map<K, V>& map,
+                                     const K& value) {
+  auto it = map.find(value);
+  if (it == map.end()) {
+    return absl::InvalidArgumentError("No table with given name");
```

Comment:
Ten błąd jest taki sobie, bo zazwyczaj szukamy po id, bardziej mi pasuje "No value for given key" i drugi argument nazwałbym 'key'